packages/lesmis/src/views/DashboardView.js

```diff
@@ -360,7 +368,7 @@ const VillageView = ({ vitals }) => {
           isLoading={vitals.isLoading}
         />
       </VitalsSection>
-      <MiniMap entityCode={vitals.code} />
+      <PhotoOrMap vitals={vitals} />
```

Comment:
Is it right that we will always show the photo over the map if it exists? Even for districts?

Reply:
I reckon so! I thought about this and decided...if someone's decided that any given entity, of any type, deserves a photo, we should show it